posts/2024/2024_08_22_inigo_final_report.rst

```diff
@@ -0,0 +1,309 @@
+* Integrate the replicated unconditional AutoEncoder architecture and sampling algorithm into DIPY to generate synthetic human tractograms, after training it with extensive human tractogram data.
+* Investigate how to condition the AutoEncoder architecture on scalar and categorical variables to support conditional tractogram generation on age and neurodegeneration status.
+* Implement a conditional AutoEncoder architecture that can generate synthetic tractograms conditioned on age and neurodegeneration status.
+
+|modified-objectives-title|
+---------------------------
+
+* Investigate the use of Variational AutoEncoders (VAE) for unconditional tractogram generation based on the FINTA architecture.
+* Investigate the use of Conditional Variational AutoEncoders (CVAE) for conditional tractogram generation based on the FINTA architecture.
+* Investigate the possibility to condition the tractogram generation on the fiber bundle for additional control over the process and the data generation, using Adversarial AutoEncoders with attribute-based regularization.
+
+|objectives-completed-title|
+----------------------------
+
+* **Literature Review on synthetic tractography generation using AutoEncoders**. Main inspirations: 
+
+  * The `FINTA <https://doi.org/10.1016/j.media.2021.102126>`_ and `GESTA <https://doi.org/10.1016/j.media.2023.102761>`_ papers because they provide a relatively simple AE architecture with an open source sampling algorithm, easy to reuse.
+
+  * `Variational AutoEncoders for Regression <https://doi.org/10.1007/978-3-030-32245-8_91>`_, which provided a good starting point for conditional Variational AutoEncoders with direct application to brain aging, related to the project's objectives.
+
+  * `Attribute-based Regularization of latent spaces for variational AutoEncoders <https://doi.org/10.1007/s00521-020-05270-2>`_ and `Adversarial AutoEncoders <http://arxiv.org/abs/1511.05644>`_, which inspired the use of Attribute-based regularization for conditioning on a continuous variable (age); and the use of adversarial training for conditioning on categoric variables (fiber bundle or neurodegeneration status), respectively.
+
+* **Replicated the** `FINTA <https://doi.org/10.1016/j.media.2021.102126>`_ **architecture originally implemented in PyTorch (vanilla AutoEncoder, not variational) using TensorFlow2+Keras.** 
+  Validated the results using the FiberCup dataset. The model is found in the ``ae_model.py`` module of the `TractoEncoder GSoC <https://github.com/itellaetxe/tractoencoder_gsoc>`_ repository. The architecture can be summarized as in the following image:
+
+  .. image:: /_static/images/gsoc/2024/inigo/inigo_vanilla_autoencoder.png
+    :alt: AE architecture diagram.
+    :align: center
+    :width: 800
```

Comment:
Make all images in this post - width to 600 and add the class we defined earlier.

```
:width: 600
:class: custom-margin
```

This will provide a cleaner look

Reply:
done, ty